etc/ui-config/mh_default_org/paella/config.json

```diff
@@ -348,7 +348,7 @@
         "enabled": true
       },
       "es.upv.paella.opencast.userTrackingSaverPlugIn": {
-        "enabled": true
+        "enabled": false
```

Comment:
This is only indicating that the plugin is loaded, but the plugin checks that the user tracking is enabled using the rest endpoint `/usertracking/detailenabled` as documentation says.

https://github.com/opencast/opencast/blob/a6d60b86d71f6ed55f19d860198c80188315c9f5/modules/engage-paella-player/src/main/paella-opencast/plugins/es.upv.paella.opencast.userTrackingSaverPlugIn/mh_usertracking_saver.js#L25-L39

Paella is using the value of the `org.opencastproject.usertracking.detailedtrack` property of the `etc/org.opencastproject.usertracking.impl.UserTrackingServiceImpl.cfg` file.

Maybe it's better to disable it in `etc/org.opencastproject.usertracking.impl.UserTrackingServiceImpl.cfg` file.
https://github.com/opencast/opencast/blob/db8f97a4d5ba21dfb89add6cadf572430cc4780b/etc/org.opencastproject.usertracking.impl.UserTrackingServiceImpl.cfg#L1-L5

If it is disabled by default, please modify also the documentation:
https://github.com/opencast/opencast/blob/db8f97a4d5ba21dfb89add6cadf572430cc4780b/docs/guides/admin/docs/configuration/user-statistics.and.privacy.md#L19-L21